### DISCUSSION TITLE
Failed to execute 'subscribe' on 'PushManager': Subscription failed - no active Service Worker error in flutter web

### DISCUSSION THREAD

**Discussion by FarzaneGhb** [firebase/flutterfire#11900] [Q&A]
Im using firebase messaging in my flutter app.
 Why do I get this error: "Error: AbortError: Failed to execute 'subscribe' on 'PushManager': Subscription failed - no active Service Worker " when I run my Flutter app for the first time on Chrome browser, but if I restart the app or refresh the browser page, the error disappears?
and how should i solve that?

**Comment by shrijanRegmi** [comment]
Same here

**Comment by dazenk** [comment]
The same thing happens to me in my flutter app.

**Comment by LostInDarkMath** [comment]
I have the same issue. Any ideas?
